src/Controls/src/Core/Shapes/LineGeometry.cs

```diff
@@ -32,5 +32,13 @@ public Point EndPoint
 			set { SetValue(EndPointProperty, value); }
 			get { return (Point)GetValue(EndPointProperty); }
 		}
+
+		public override PathF PathForBounds(Graphics.Rectangle rect)
+		{
+			var path = new PathF();
+			path.Move((float)StartPoint.X, (float)StartPoint.Y);
+			path.LineTo((float)EndPoint.X, (float)EndPoint.Y);
+			return path;
```

Comment:
Geometry doesn't have a concept of bounds before being asked for the path.  This is kind of strange, so we just have to basically assume the given fixed values are correct?  Should we do any bounds checking here?